samples/openthread/cli/README.rst

```diff
@@ -64,6 +64,21 @@ See `Testing diagnostic module`_ section for an example.
 .. note::
     If you disable the :kconfig:option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` feature set, you can enable the diagnostic module with the :kconfig:option:`CONFIG_OPENTHREAD_DIAG` Kconfig option.
 
+Additionally for the ``nrf52840dongle_nrf52840`` build target, diagnostic GPIO commands have been enabled by ``openthread_config`` node in :file:`boards/nrf52840dongle_nrf52840.overlay`.
+This node has been configured for P0.19 pin, which is connected to reset on nRF52840 Dongle.
```

Comment:
```suggestion
For the ``nrf52840dongle_nrf52840`` build target, diagnostic GPIO commands can also be used.
They are enabled in the ``openthread_config`` node in the :file:`boards/nrf52840dongle_nrf52840.overlay` file, with the node configured for the **P0.19** pin, which is connected to reset on the nRF52840 Dongle.
```
Should "reset" be "the **RESET** button" here?

Reply:
Maybe "the RESET pin" or "the RESET signal"? Button itself is connected to RESET path so it doesn't sound intuitive to me
![image](https://github.com/nrfconnect/sdk-nrf/assets/115064000/34334225-fc44-4629-88e3-1b2d8b0b54bd)

Reply:
Yeah, that looks like "the **RESET** pin" would be correct. I did take a quick look at the diagram, but didn't notice it there.